graphql/util.go

```diff
@@ -1406,3 +1406,20 @@ func hasProjectPermission(ctx context.Context, resource string, next graphql.Res
 	}
 	return nil, Forbidden.Send(ctx, fmt.Sprintf("user %s does not have permission to access settings for the project %s", user.Username(), resource))
 }
+
+// getValidTaskStatusesFilter returns a slice of task statuses that are valid and are searchable.
+// It returns an empty array if all is included as one of the entries
+func getValidTaskStatusesFilter(statuses []string) []string {
+	filteredStatuses := []string{}
+	if utility.StringSliceContains(statuses, evergreen.TaskAll) {
+		return filteredStatuses
+	}
+	filteredStatuses = utility.StringSliceIntersection(evergreen.TaskStatuses, statuses)
+	for _, status := range statuses {
```

Comment:
Did you purposefully leave this for loop here? StringSliceIntersection already handled this so now you've added all statuses twice

Reply:
oops